hack/builder/Dockerfile

```diff
@@ -62,9 +62,12 @@ RUN set -x && \
     go get -v github.com/golang/protobuf/protoc-gen-go@1643683 && \
     go get -v k8s.io/code-generator/cmd/client-gen@v0.16.4 && \
     go get -v github.com/securego/gosec/v2/cmd/gosec@0ce48a5 && \
-    go get -v sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.0 && \
     go clean -cache -modcache
 
+RUN git clone https://github.com/xpivarc/controller-tools.git && \
+    cd controller-tools/cmd/controller-gen && git checkout string-fix && \
+    source /etc/profile.d/gimme.sh && go install 
+
```

Comment:
i see you made a bug fix that you're trying to get merged into controller-tools https://github.com/kubernetes-sigs/controller-tools/issues/507

What field or fields does this bug fix impact in our API?

Reply:
Fields that have 'string' in json tag. [i.e](https://github.com/kubevirt/kubevirt/pull/4396/files?file-filters%5B%5D=.go&file-filters%5B%5D=.sh&file-filters%5B%5D=No+extension#diff-194ad1a8d11fa7160d03e60a5ab73fef10057b639474874d7f5ee32040b40cb3R270).
`integer` -> `string` so validation will not fail.

Reply:
i see. so would this make both  `resource.requests.CPU = 2` and `resource.requests.CPU = "2"` work? where the latter would fail without the change?

Reply:
What I don't understand, if we want both to be valid, why don't we use `IntOrString` type?

Reply:
Only the latter succeeds with this change. If we want both to work we need `IntOrString` as Roman suggested. Maybe I misunderstood the API, do you think we want both?